errors/template.am

```diff
@@ -48,6 +48,5 @@ ERROR_TEMPLATES = \
 	templates/ERR_TOO_BIG \
 	templates/ERR_UNSUP_HTTPVERSION \
 	templates/ERR_UNSUP_REQ \
-	templates/ERR_URN_RESOLVE \
```

Comment:
This PR does not update PO/POT files based on an earlier [recommendation](https://github.com/squid-cache/squid/pull/1092#discussion_r923985081). If those files should be updated to reflect ERR_URN_RESOLVE removal, please let me know, and we will update them (you can even preview most of those changes in earlier branch commit c00c79a4 that was later reverted).

Reply:
These templates are used to generate the "langpack" releases which get installed for use by much older versions of Squid. The template file needs to be retained until no supported version of Squid tries to load it on startup.

This is also why the ERR_ESI remains.

Reply:
> The template file needs to be retained 

Restored at 0f42fbd.